python/GafferTest/TestCase.py

```diff
@@ -346,9 +346,13 @@ def assertModuleDoesNotImportUI( self, moduleName ) :
 			f.write( "import sys\n" )
 			f.write( "assert( 'GafferUI' not in sys.modules )\n" )
 
-		subprocess.check_call( [ "gaffer" if os.name != "nt" else "gaffer.cmd", "python", script ] )
+		subprocess.check_call( [ self.gafferExecutable(), "python", script ] )
 
 	def assertFloat32Equal( self, value0, value1 ) :
 
 		from GafferTest import asFloat32
 		self.assertEqual( asFloat32( value0 ), asFloat32( value1 ) )
+
+	@staticmethod
+	def gafferExecutable() :
+		return "gaffer" if os.name != "nt" else "gaffer.cmd"
```

Comment:
Those both sound good to me. I added those two functions and fixed up the tests to use them.

I didn't change the existing uses of `os.path.expandvars()` to use `Gaffer.rootPath()` because all of them that I see in tests are using a full string with forward slashes, not a concatenation of `pathlib.Path` items. Since they are already in our preferred format, is it better to leave them alone, or convert them to `pathlib.Path`?

Reply:
Those are squashed into 276ffc5591dd8aaec408524285fe4b3039b1e505